package/README.md

```diff
@@ -0,0 +1,103 @@
+## Description
+
+CLI Application that lists objects with given type from OpenAPI document.
+
+Note - Even though all types have been seperately tested, there is a chance of errors. If such a thing happens please contact the authors.
+
+## Installation
+```
+npm i -g openapi-type-finder
+```
+
+## Target
+```
+    openapi-type-finder -u <url> -t <type> (optional) --no-resolve
+```
+Use --no-resolve if you want the references in the document to not be resolved, though it may lead to inacurrate behaviour.
+
+Test
+
+Input url: https://api.apis.guru/v2/specs/xkcd.com/1.0.0/openapi.yaml
+
+Input type: responses
+
+Output:
+
+```
+{
+    "content": {
+        "*/*": {
+            "schema": {
+                "$ref": "#/components/schemas/comic"
+            }
+        }
+    },
+    "description": "OK"
+}
+```
+***
+
+# Types Done:
```

Comment:
I like things being documented but these can have descriptions

Reply:
Description about the checklist - the purpose, etc., but this can wait - not a must

Reply:
Fixed